### ISSUE TITLE
Aural proc linking problem

### ISSUE THREAD

**Issue by Sciss** [Sciss/SoundProcesses#18]
With first impl of audition tool in Mellite, we find a non-deterministic problem with sound not being wired up. It appears that this has to do with hashes in a map in `Transport` if `play` is issued only after all objects have been added. The below OSC dump shows the problem.

The constellation is

```
t.addObject(global1)
t.addObject(global2)
t.addObject(x)
t.play()
```

where `x` feeds into one the globals.
It is not obvious why the BROKEN variant doesn't sound, as `mapan` should work as `InFeedback`, i.e. irrespective of node order.

```
--------------------------------------- WORKS
s: [ #bundle, <now>,
  [ "/s_new", "Out-L_1", 1112, 0, 1, "$at_pan", -1.0 ],
  [ "/n_mapan", 1112, "$i_in", 3, 1 ],
  [ "/s_new", "Out-L_1", 1113, 0, 1, "$at_pan", 1.0 ],
  [ "/n_mapan", 1113, "$i_in", -1, 1 ],
  [ "/s_new", "(504,513)_0", 1114, 0, 1, "$at_fade-in", [ 0.01, 1.0, 0.0, 0.0 ], "$at_fade-out", [ 3.0, 1.0, 0.0, 0.0 ], "$o_out", 4.0 ],
  [ "/s_new", "a-ln1_2", 1115, 2, 1112, "in", 4.0, "out", 3.0 ] ]
r: [ "/n_go", 1112, 1, -1, -1, 0 ]
r: [ "/n_go", 1113, 1, -1, 1112, 0 ]
r: [ "/n_go", 1114, 1, -1, 1113, 0 ]
r: [ "/n_go", 1115, 1, 1113, 1112, 0 ]
s: [ #bundle, <now>,
  [ "/n_free", 1115 ],
  [ "/n_free", 1112 ],
  [ "/n_free", 1113 ],
  [ "/n_free", 1114 ] ]
r: [ "/n_end", 1115, 1, 1113, 1112, 0 ]
r: [ "/n_end", 1112, 1, 1113, -1, 0 ]
r: [ "/n_end", 1113, 1, 1114, -1, 0 ]
r: [ "/n_end", 1114, 1, -1, -1, 0 ]

--------------------------------------- BROKEN
s: [ #bundle, <now>,
  [ "/s_new", "(504,513)_0", 1116, 0, 1, "$at_fade-in", [ 0.01, 1.0, 0.0, 0.0 ], "$at_fade-out", [ 3.0, 1.0, 0.0, 0.0 ], "$o_out", 4.0 ],
  [ "/s_new", "Out-L_1", 1117, 0, 1, "$at_pan", 1.0 ],
  [ "/n_mapan", 1117, "$i_in", -1, 1 ],
  [ "/s_new", "Out-L_1", 1118, 0, 1, "$at_pan", -1.0 ],
  [ "/n_mapan", 1118, "$i_in", 2, 1 ],
  [ "/s_new", "a-ln1_2", 1119, 2, 1118, "in", 4.0, "out", 2.0 ] ]
r: [ "/n_go", 1116, 1, -1, -1, 0 ]
r: [ "/n_go", 1117, 1, -1, 1116, 0 ]
r: [ "/n_go", 1118, 1, -1, 1117, 0 ]
r: [ "/n_go", 1119, 1, -1, 1118, 0 ]
s: [ #bundle, <now>,
  [ "/n_free", 1119 ],
  [ "/n_free", 1116 ],
  [ "/n_free", 1117 ],
  [ "/n_free", 1118 ] ]
r: [ "/n_end", 1119, 1, -1, 1118, 0 ]
r: [ "/n_end", 1116, 1, 1117, -1, 0 ]
r: [ "/n_end", 1117, 1, 1118, -1, 0 ]
r: [ "/n_end", 1118, 1, -1, -1, 0 ]
```

**Comment by Sciss** [comment]
Suspecting that this was fixed in c79143df570babc37af984e5ab4413dda058c595 - because we have `a_ln` synths here which did _not_ use `InFeedback`.

**Comment by Sciss** [comment]
Closing now -- most likely outdated.
